src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -129,18 +128,10 @@ public void validate(FailureCollector collector) {
         try {
           new SimpleDateFormat(suffix);
         } catch (IllegalArgumentException e) {
-          collector.addFailure("Invalid suffix.", "Ensure provided suffix is valid.")
+          collector.addFailure("Invalid suffix : " + e.getMessage(), null)
             .withConfigProperty(NAME_SUFFIX).withStacktrace(e.getStackTrace());
         }
       }
-      if (!containsMacro(NAME_FORMAT)) {
-        try {
-          getFormat();
```

Comment:
doesn't this still need to be called?

Reply:
When would it be invalid? since its not macro enabled.